pkg/workload/workload.go

```diff
@@ -394,6 +390,18 @@ func SetRequeuedCondition(wl *kueue.Workload, reason string, message string) {
 	apimeta.SetStatusCondition(&wl.Status.Conditions, condition)
 }
 
+// UnsetRequeuedCondition sets the WorkloadRequeued condition to false
+func UnsetRequeuedCondition(wl *kueue.Workload, reason string, message string) {
+	condition := metav1.Condition{
+		Type:               kueue.WorkloadRequeued,
+		Status:             metav1.ConditionFalse,
+		Reason:             reason,
+		Message:            api.TruncateConditionMessage(message),
+		ObservedGeneration: wl.Generation,
+	}
+	apimeta.SetStatusCondition(&wl.Status.Conditions, condition)
+}
```

Comment:
Could you commonize the `SetRequeuedCondition` and the `UnSetRequeuedCondition` functions?
Also, could you re-title the `UnSetRequeuedCondition` to something like the `DisabledRequeuedCondition` since the `UnsetRequeuedCondition` never unset the `Requeued` condition?
@alculquicondor WDYT about the function name, `DisabledRequeuedCondition`? Any recommended name?

```golang
func SetRequeuedCondition(wl *kueue.Workload, reason, message string) {
	updateRequeuedCondition(wl, reason, message, metav1.ConditionTrue)
}

func DisableRequeuedCondition(wl *Workload, reason, message string) {
	updateRequeuedCondition(wl, reason, message, metav1.ConditionFalse)
}

func updateRequeuedCondition(wl *Workload, reason, message string, status metav1.ConditionStatus) {
	condition := metav1.Condition{
		Type:               kueue.WorkloadRequeued,
		Status:             status,
		Reason:             reason,
		Message:            api.TruncateConditionMessage(message),
		ObservedGeneration: wl.Generation,
	}
	apimeta.SetStatusCondition(&wl.Status.Conditions, condition)
}
```

Reply:
Done